Gems/Atom/Feature/Common/Assets/Materials/Pipelines/LowEndPipeline/TintedTransparent_StandardLighting.shader.template

```diff
@@ -0,0 +1,52 @@
+{
+    "Source" : "INSERT_AZSL_PATH_HERE",
+
+    "DepthStencilState" :
+    {
+        "Depth" :
+        {
+            "Enable" : true,
+            "CompareFunc" : "GreaterEqual",
+            "writeMask": "Zero"
+        },
+        "Stencil" :
+        {
+            "Enable" : true,
+            "ReadMask" : "0x00",
+            "WriteMask" : "0xFF",
+            "FrontFace" :
+            {
+                "Func" : "Always",
+                "DepthFailOp" : "Keep",
+                "FailOp" : "Keep",
+                "PassOp" : "Replace"
+            }
+        }
+    },
+    
+    "GlobalTargetBlendState" : 
```

Comment:
Note to reviewers: GlobalTargetBlendState is different between the **[tinted]transparent** and **opaque** .shader.template files